### DISCUSSION TITLE
Translation issues

### DISCUSSION THREAD

**Discussion by dragomano** [statamic/cms#8134] [Troubleshooting]
So, when I translated Statamic's UI into Russian, I noticed these issues.

Create any field area

1. In the title, we see "Переключатель Поле", and "Переключатель Тип поля" (English: "Toggle Field", and "Toggle Fieldtype"). Is there any way to make these words swap places in the Russian translation? Desired result: "Поле: Переключатель", and "Тип поля: Переключатель". Is it possible to create some kind of template for such combined titles? For example, '$1$s: %2%s'.
![sshot-3](https://github.com/statamic/cms/assets/229402/16d6cea1-a4d3-4cc8-a5cb-f573e200d7d2)

2. Then, when we are filling in the title of the field, a translation of the field type is substituted, and accordingly the "Handle" field is automatically filled with the generated slug. For example, if the title of the field specifies "Textarea Field", then in the "Handle" field will be "tekstovaya_oblast_pole". Is it possible to make, first, that in the header the word "Поле" stood in front: "Поле Текстовая область", and secondly, in the "Handle" field would the text be generated based on the original English field names? That is: "textarea_field" instead of "tekstovaya_oblast_pole".
![sshot-1](https://github.com/statamic/cms/assets/229402/837c9e54-de98-4159-bc46-749295bff480)

**Comment by jasonvarga** [comment]
Yes, nice finds. Fixed in #8141
